docs/references/creating-a-release.md

```diff
@@ -10,41 +10,88 @@ We must do due diligence in testing our packages before any new major/minor or b
 
 See the [release checklist](../references/release-checklist.md) for tasks that must be completed before releasing.
 
-### Release branches
+## Release workflow
 
-A release branch is created for each new beta and stable release (alpha version are excluded from this). The release branch is a short lived branch used to ensure high release quality.
+Both beta and stable release follow a two step workflow, aided by github actions.
 
-While the release branch is active, some changes might be merged into the branch (for bug fixes, features deemed necessary for the release). PRs into the release branch should follow this process when possible:
+### Step 1: Create a Pre-release branch
```

Comment:
nit: can you add just below this the purpose of a prerelease branch. It would be good to let the reader know what is overall happening in this step and why

```
A prerelease branch is for x,y,z. It bumps packages, etcx. etc. and is merged back to main before creating the release branch.
```

Or alternatively above this state the steps and why we do them
```
1. Create Pre-Release Branch
  A prerelease branch is for x,y,z. It bumps packages, etcx. etc. and is merged back to main before creating the release branch.
2. Merge release branch back to main
  This is to ensure blah blah
4. Create release branch off main
  This is the branch the release will be created off... etc.
6. etc.
```